system/settings/settings.xml

```diff
@@ -1411,7 +1411,7 @@
         </setting>
         <setting id="pvrplayback.confirmchannelswitch" type="boolean" label="19281" help="36231">
           <level>1</level>
-          <default>false</default>
+          <default>true</default>
```

Comment:
It is used exclusively with the channel surfing feature (pressing up/down key while in fullscreen PVR / music vis PVR). If you want channelchange without confirmation you can always use channel up/channel down keys on your remote.